src/main/kotlin/io/zerobase/smarttracing/features/organizations/OrganizationsResource.kt

```diff
@@ -119,21 +118,8 @@ class OrganizationsResource(
         dao.setMultiSite(OrganizationId(id), hasMultipleSites)
     }
 
-    @Path("/{orgId}/sites/{siteId}/scannables")
-    @POST
-    @Creator
-    fun createScannable(@PathParam("orgId") orgId: String, @PathParam("siteId") siteId: String,
-                        request: CreateScannableRequest): IdWrapper {
-        val type = request.type
-        val singleUse = request.singleUse
-
-        if (!scanTypes.contains(type)) {
-            val res = Response.status(Response.Status.BAD_REQUEST)
-                .entity("Not a valid type please check /models/scannable-types")
-                .build()
-            throw WebApplicationException(res)
-        }
-
-        return dao.createScannable(OrganizationId(orgId), SiteId(siteId), type, singleUse).let(::IdWrapper)
+    @Path("/{orgId}/sites/{siteId}")
```

Comment:
I'm definitely ignorant to Kotlin, but if I'm understanding this - the new path is /{orgID}/sites/{siteID} and the scannable ID is within the request body

Reply:
tl;dr - no, this is just JAX-RS (standard java rest api spec) stuff. none of the api endpoints have changed in this refactor. 

technical details:
the `@Path` annotations are combined by the framework. they can be placed on the class or the method. root resource classes require a `@Path` annotation on the class, but sub-resources do not. a "sub-resource" is a class that contains JAX-RS annotated methods but does not have a class-level `@Path` annotation and is returned by a method on another JAX-RS annotated class. it allows you to split up your resources more to be more specific in what they handle, and it lets you do extra calculations before delegating down. for example, we could do a quick check to validate that the organization and site actually exist before we delegate to the site specific methods.